compiler/modules/front/codeGen.bal

```diff
@@ -1229,6 +1245,108 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Environment env, s:Ex
     panic err:impossible("unrecognized expression type in code gen: " +  s:exprToString(expr));
 }
 
+function intOperand(CodeGenContext cx, bir:Operand operand, Range subExprRange) returns bir:IntOperand|CodeGenError {
+    if operand is bir:IntOperand {
+        return operand;
+    }
+    return cx.semanticErr("expected an int operand", subExprRange);
+}
+
+function exprRange(s:Expr expr) returns Range {
```

Comment:
There's already an `s:range` function to do this.

Reply:
fixed